docs/tutorials/tensorflow_inference_and_training.md

```diff
@@ -0,0 +1,152 @@
+# Tutorial: Optimize TensorFlow Models with BladeDISC
+
+In this tutorial, we show how to optimize TensorFlow models with BladeDISC for
+both inference and training. Users only need to add two lines of code to
+optimize the model just-in-time for the examples in this tutorial.
+
+These packages are required before going through the tour:
```

Comment:
Users don't need to install these two packages if they have installed BladeDISC as `Install BladeDISC With Docker` docs.

Reply:
Done. Thanks.